scripts/tasks/fluentui-publish.ts

```diff
@@ -1,33 +1,163 @@
+  });
+  if (result.status) {
+    throw new Error(result.error?.stack || `'${command} ${args.join(' ')}' failed with status ${result.status}`);
+  }
+  return result.stdout;
+};
+
+export function fluentuiPostPublishValidation() {
   return function() {
-    const fluentRoot = path.resolve(findGitRoot(), 'packages', 'fluentui');
-    const lernaPublishArgs = [
-      'lerna',
-      'publish',
-      "--tag-version-prefix='@fluentui/react-northstar_v'", // HEADS UP: also see yarn stats:save in azure-pipelines.perf-test.yml
-      '--no-git-reset',
-      '--force-publish',
-      '--registry',
-      argv().registry,
-      bumpType,
-    ];
-
-    logger.info(`Running this command: yarn ${lernaPublishArgs.join(' ')}`);
-
-    const result = spawnSync('yarn', lernaPublishArgs, {
-      cwd: fluentRoot,
-      shell: true,
-      stdio: 'inherit',
-    });
+    const gitRoot = findGitRoot();
 
-    if (result.status) {
-      throw new Error(result.error?.stack || `lerna publish failed with status ${result.status}`);
+    const branch = execCommandSync(gitRoot, 'git', ['branch', '--show-current']);
+    execCommandSync(gitRoot, 'git', ['fetch', 'origin']);
+    execCommandSync(gitRoot, 'git', ['reset', '--hard', `origin/${branch}`]); // sometimes lerna add gitHead in package.json after release
+
+    // sync fluent version
+    execCommandSync(gitRoot, 'yarn', ['syncpack:fix']);
+    const gitStatus = execCommandSync(gitRoot, 'git', ['status']);
+    if (!gitStatus.includes('nothing to commit, working tree clean')) {
+      execCommandSync(gitRoot, 'git', ['add', `\*package.json`]);
+      execCommandSync(gitRoot, 'git', ['commit', '-m', `"chore: fix dependencies after react-northstar release"`]);
+      execCommandSync(gitRoot, 'git', ['push']);
     }
+
+    // make sure there's no more than one fluent versions
+    execCommandSync(gitRoot, 'yarn', ['syncpack:list']);
+  };
+}
+
+// pack all public fluent ui packages, used by ci to store nightly built artifacts
+const PUBLIC_FLUENT_PACKAGES = [
+  'ability-attributes',
+  'accessibility',
+  'code-sandbox',
+  'docs-components',
+  'react-bindings',
+  'react-builder',
+  'react-component-event-listener',
+  'react-component-nesting-registry',
+  'react-component-ref',
+  'react-context-selector',
+  'react-icons-northstar',
+  'react-northstar-emotion-renderer',
+  'react-northstar-fela-renderer',
+  'react-northstar-styles-renderer',
+  'react-northstar',
+  'react-proptypes',
+  'react-telemetry',
+  'state',
+  'styles',
+];
+
+const TODAY = new Date().toISOString().split('T')[0]; // yyyy-mm-dd
+
+const replaceDepVersionWithNightlyUrl = (packageRoot, packageName) => {
+  const packageJson = require(`@fluentui/${packageName}/package.json`);
+  const dependencies = packageJson.dependencies || {};
+  let hasFluentDependency = false;
+  for (const key of Object.keys(dependencies)) {
+    if (key.startsWith('@fluentui/') && PUBLIC_FLUENT_PACKAGES.includes(fluentPackageName)) {
```

Comment:
```suggestion
    if (PUBLIC_FLUENT_PACKAGES.includes(fluentPackageName)) {
```

As we will use Lerna's output, we technically don't need this check